examples/pipeline_tacotron2/text/numbers.py

```diff
@@ -0,0 +1,71 @@
+"""Modified from https://github.com/keithito/tacotron"""
```

Comment:
It's MIT license. Can add the copy right license notice?
https://github.com/keithito/tacotron/blob/master/LICENSE

Reply:
Thanks for pointing it out, I've uploaded it in `examples/pipeline_tacotron2/text/LICENSE`.
Thanks.

Reply:
Actually, can you copy-paste the LICENSE here directly?
That way, it is clearer which files are from `keithito/tacotron` and when we add something new to the same directory, it will be ambiguous which one belongs to the MIT license.

Reply:
Same goes to `text_preprocessing.py`.

Reply:
Similar to https://github.com/pytorch/audio/pull/1625/files#diff-446b770bff29efd0f406a17635f004c65a136ace539bd3d4eea923e14de8915dR1

Reply:
Thanks for pointing it out, I've updated `examples/pipeline_tacotron2/text/numbers.py` and `examples/pipeline_tacotron2/text/text_preprocessing.py`.